OKR/PROJECT.md

```diff
@@ -2,8 +2,29 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
-## 2018 Q4
+## 2019 Q1
 
+- **The IPFS Org gets a fully dedicated Package Managers Working Group**
+  - `P0` - @daviddias - A Package Manager Working Group Roadmap is created
+  - `P1` - @??? - Onboard 1+ contributors to Package Managers Working Group
+- **Support and guide the IPFSConf planning and Content Curation**
+  - `P0` - @daviddias - The IPFSConf is scheduled, announced and there is a CFP running. The IPFSConf Org Team is supported by the IPFS org.
+  - `P0` - @momack2 - The IPFS Org (Working Groups & Core Contributors) has a unified plan with goals for IPFSConf
+- **The IPFS Working Groups feel supported and they have the information they need to make the best decisions**
+  - `P0` - @mikeal - We gather, process and present metrics for the IPFS Project  
+  - `P1` - @parkan - Ship the first IPFS User Quarterly Report. It documents achievements and demos of new capabilities
+  - `P1` - @momack2 - Working group roadmaps are solidified, presented, and used to chart Q1 and Q2 planning
```

Comment:
```suggestion
  - `P1` - @momack2 - Working group roadmaps are solidified, presented, and used to chart Q1 and Q2 planning
  - `P1` - @???  - The IPFS Project publishes a Q1 Project Report with WG updates and demos
```

We said we were going to do Project Reports. David - you have a lot of P0s so I'm happy to work with Jenn to drive this forward with your feedback if you don't have bandwidth (but happy for you to own it if you prefer).

Reply:
We said these were different things in our Roadmap: User reports about user needs and use cases vs Project reports about new features shipped, demos, etc.

Reply:
We decided we'd drop the second report (project updates) and simply work with @renrutnnej to make end-of-quarter newsletters include more of a look back on the launches/features/accomplishments of the past quarter. See comment here: https://github.com/ipfs/newsletter/issues/74#issuecomment-457718255